lcc_comchain_base/views/account_invoice_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="view_account_invoice_customer_conchain_form" model="ir.ui.view">
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//sheet" position="before">
+                <div groups="account.group_account_invoice" class="alert alert-info" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('comchain_amount_to_credit','=',0)]}">
+                    You have <bold><field name="comchain_amount_to_credit"/></bold> numeric currency to credit for this customer. Please go to mobile app to validate credit.
```

Comment:
"Please use" instead of "Please go to" seems more natural. And the last "credit" would maybe clearer if called "credit requests".